data_ingestion/app.py

```diff
@@ -61,16 +59,32 @@ def send_points(self, points_by_bucket):
         for bucket in points_by_bucket:
             self.write_api.write(bucket, self.org, points_by_bucket[bucket])
```

Comment:
We need to change the bucket config in the pipeline to remove the alerts, telem, events buckets. Can you remove those configs and upload one config for just an Iris bucket.

Reply:
We will also need to fix mission api to send alerts to the same bucket as well, we can address this in a separate PR.

Reply:
WIP

Reply:
Resolved. Please check the PR.